README.md

```diff
@@ -314,9 +314,10 @@ For more information on limitations and recommendations, please refer to [IAM da
 For more information, please refer to [Creating a database account using IAM authentication](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.DBAccounts.html#UsingWithRDS.IAMDBAuth.DBAccounts.PostgreSQL).
 
 #### Using AWS IAM Database Authentication
-| Parameter       | Value           | Default Value      | Description  |
-| ------------- |:-------------:|:-------------:| ----- |
-|`useAwsIam` | Boolean | `false` | Set to `true` to use AWS IAM database authentication. |
+| Parameter                       |  Value  | Default Value | Description                                                                                                                                                                                                                                                                                                  |
+|---------------------------------|:-------:|:-------------:|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `useAwsIam`                     | Boolean |    `false`    | Set to `true` to use AWS IAM database authentication.                                                                                                                                                                                                                                                        |
+| `authenticationPluginClassName` | String  |     null      | Fully qualified class name of the class implementing the AuthenticationPlugin interface. If this is null, the password value in the connection properties will be used. Set this to the AWSIamAuthenticationPlugin class name to use AWS IAM database authentication (example available in the sample code). |
```

Comment:
`useAwsIam` is unused/useless, please remove.
& wouldn't really call the `authenticationPluginClassName` a parameter for AWS IAM, could you move it to outside of the table / remove the table?